docker/README.md

```diff
@@ -61,16 +61,43 @@ Common configuration template is contained in `docker/.env.template`. Developers
 
 The template contains an already working configuration to get started.
 
-## Local environment secrets
+## Local environment secrets (not required)
 
 By default, the local setup will build the ckan and drupal image frontends using free version of fontawesome.
 
 To use the pro version, you must provide a valid .npmrc file either via BuildKit secrets or Dockerfile build-time ARG. BuildKit secrets are meant for production image builds because they provide a secure way to use secrets during image builds. For local development, use the build-time ARG option instead because it is supported in docker-compose builds.
 
-### Example docker-compose.override.yml for local development
+## Steps for setting up the local dev environment
 
-Create a file `docker-compose.override.yml` to the `docker` directory and populate its contents with the example below and edit your .npmrc file contents in it appropriately.
+If you're not on MacOS, make sure you have enabled Buildkit for docker and docker-compose as mentioned above for your system.
 
+0. Make sure you have copied the `docker/.env.template` to `docker/.env` as mentioned above.
+
+1. Create a file `docker-compose.override.yml` to the `docker` directory and populate its contents with the example below (next topic)
+
+2. Clone the following repos to the ../../ path: opendata-ckan, opendata-drupal, opendata-solr, opendata-nginx
```

Comment:
Maybe it would be better to just describe structure without relative paths (and use root instead of `../..`):
```
- root
  - opendata (https://github.com/vrk-kpa/opendata)
  - opendata-ckan (https://github.com/vrk-kpa/opendata-ckan)
  - opendata-drupal (https://github.com/vrk-kpa/opendata-drupal)
  - opendata-nginx (https://github.com/vrk-kpa/opendata-nginx)
  - opendata-solr (https://github.com/vrk-kpa/opendata-solr)
```